examples/tables.ipynb

```diff
@@ -329,17 +347,17 @@
     "count = 0\n",
     "for col_name in inplace.columns[:20]:\n",
     "    col = inplace[col_name]\n",
-    "    print(col.dataframe.head(1))\n",
+    "    print(col.to_pandas.head(1))\n",
     "    count += 1\n",
     "print(f\"{count} cols in total time: {time.perf_counter() - start: .1f} s\")"
    ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "test_fmu-sumo",
+   "display_name": "venv",
```

Comment:
I assume this shouldn't be like this. But I wanted to update notebook with latest changes???

Reply:
It will be a bit noisy... Just make sure you clear all results from the notebook (restart kernel and clear output) before you commit and push.